composer/models/tasks/classification.py

```diff
@@ -49,34 +51,44 @@ class ComposerClassifier(ComposerModel):
         from composer.models import ComposerClassifier
 
         pytorch_model = torchvision.models.resnet18(pretrained=False)
-        model = ComposerClassifier(pytorch_model)
+        model = ComposerClassifier(pytorch_model, num_classes=1000)
     """
 
     num_classes: Optional[int] = None
 
     def __init__(self,
                  module: torch.nn.Module,
+                 num_classes: Optional[int] = None,
                  train_metrics: Optional[Union[Metric, MetricCollection]] = None,
                  val_metrics: Optional[Union[Metric, MetricCollection]] = None,
                  loss_fn: Callable = soft_cross_entropy) -> None:
         super().__init__()
 
+        self.module = module
+        self._loss_fn = loss_fn
+
+        self.num_classes = num_classes
```

Comment:
which should we take if `num_classes` is passed in and an attr on the module? should we warn that both were set and say which one we're using? could be some confusing errors other wise